src/config.ts

```diff
@@ -1,4 +1,5 @@
 import {CLIError, error, exit, warn} from '@oclif/errors'
+import WSL = require('is-wsl')
```

Comment:
though the linter doesn't catch this, can you move this below the imports and format as:

`const WSL = require('is-wsl')`

Reply:
done; wasn't sure if you wanted it just after the imports or not so I moved it to a section above the primary class with similar kinds of utility functions